### ISSUE TITLE
sqrt(-1) is undef

### ISSUE THREAD

**Issue by thertp** [numworks/epsilon#79]
After updating my device (with stock firmware) to firmware version 1.1.0 (git head points to commit 88630addd4be9dd98d4d5ef6cfcf9bb5488b45be), now, if I do:
```
i^2
          -1

sqrt(Ans)
          Undef
```

I'm not familiar with the code base so for now, it's hard to tell if it's my build environment or it's a issue
in the firmware. My build environment is Debian Stretch (even cross compiler comes from Debian).
```
ii  git                            1:2.11.0-3                     amd64        fast, scalable, distributed revision control system
ii  libfreetype6-dev               2.6.3-3.2                      amd64        FreeType 2 font engine, development files
ii  bison                          2:3.0.4.dfsg-1+b1              amd64        YACC-compatible parser generator
ii  flex                           2.6.1-1.3                      amd64        fast lexical analyzer generator
ii  python                         2.7.13-2                       amd64        interactive high-level object-oriented language (default version)
ii  gcc-arm-none-eabi              15:5.4.1+svn241155-1           amd64        GCC cross compiler for ARM Cortex-A/R/M processors
ii  gdb-arm-none-eabi              7.12-6+9+b2                    amd64        GNU debugger for ARM Cortex-A/R/M processors
ii  libfltk1.3-dev                 1.3.4-4                        amd64        Fast Light Toolkit - development files
ii  clang                          1:3.8-36                       amd64        C, C++ and Objective-C compiler (LLVM based)
```

**Comment by boricj** [comment]
Probably firmware, it happens on the online simulator too.

**Comment by Ecco** [comment]
Fixed by #88
